src/server/server.cc

```diff
@@ -1839,6 +1844,26 @@ void Server::cleanupExitedWorkerThreads(bool force) {
   }
 }
 
+Status Server::forceClusterMode() {
+  std::string value;
+  auto cf = storage->GetCFHandle(engine::kPropagateColumnFamilyName);
+  rocksdb::Status check_cluster_enabled =
+      storage->Get(rocksdb::ReadOptions(), cf, rocksdb::Slice(engine::kClusterEnabledKey), &value);
+
+  if (check_cluster_enabled.IsNotFound()) {
+    Status s = storage->WriteToPropagateCF(engine::kClusterEnabledKey, std::to_string(config_->cluster_enabled));
```

Comment:
I prefer using `yes` or `no` instead of the bool value.

Reply:
Could you elaborate more on this?

Reply:
Use `yes` or `no` as the value in rocksdb instead of `0` or `1` with to_string.